modules/group/variables.tf

```diff
@@ -22,6 +22,11 @@ variable "inline_policies" {
     policy = string
   }))
   default = []
+
+  validation {
+    condition     = length(concat(var.inline_policies[*].policy)) <= 5120
```

Comment:
Would this account for AWS not counting whitespace towards that character limit

Reply:
It's mostly expecting the user to minify the policy first. And it might not be perfect when more than one policy is provided. While we could do a `jsonencode(jsondecode(...))` to minify the policy with terraform, I'm not sure how to do that over the list and still get the combined length back out.

That said, I think the way we feed our policies through the policy_document module, which uses `aws_iam_policy_document` to build up the policy document, and _that_ minifies the policy for us also. So for our particular use case, I am expecting it to be ok with the way aws handles whitespace and the character limit.